### PR TITLE
Minor edit in the CONTRIBUTING file

### PR DESCRIPTION
**Pull request by vogella**
Simplified first sentence and minor re-format of text to simplify reading in the text editor

### CODE REVIEW

**Review by HeikoKlare — COMMENTED** [review]
The proposal looks good to me. I only have one concern w.r.t. the added comma.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -31,12 +30,12 @@ The project issues and source code are maintained in [multiple GitHub repositori
...
 
-Be sure to search for existing issues before you create another one. Remember that
-contributions are always welcome!
+Be sure to search for existing issues before you create another one. 
+Remember that contributions are always welcome!
 
 ## Setting up Your Eclipse and GitHub Account
 
-Create an [Eclipse account](https://accounts.eclipse.org/) if you don't already have one. 
+Create an [Eclipse account](https://accounts.eclipse.org/), if you don't already have one.
```
[the remark this answers is not in the document]
Comment (merks) [reply CONTRIBUTING.md:38]: I'm not sure it's strictly wrong:

https://linguaholic.com/linguablog/comma-before-if/

But it does seem unnecessary and the "if" is not really "parenthetical remark"...

**Comment by akurtakov** [comment]
What's the status of this one? Please either finish or close.

**Comment by akurtakov** [comment]
It's documentation only and the suggestion has been applied.

**Pull request merged**